src/cmd/ksh93/tests/builtins.sh

```diff
@@ -996,6 +996,28 @@ then	got=$( { "$SHELL" -c '
 		"(got status $e$( ((e>128)) && print -n / && kill -l "$e"), $(printf %q "$got"))"
 fi
 
+# ==========
+# Verify that the POSIX 'test' builtin complains loudly when the '=~' operator is used rather than
+# failing silently. See //github.com/att/ast/issues/1152.
```

Comment:
missing the scheme part of the URI should be https://github.com/att/ast/issues/1152

Reply:
Thanks for pointing that out, now fixed.